lib/components/settings/widgets/settings_card.dart

```diff
@@ -208,10 +211,10 @@ class _SettingsCardState extends State<SettingsCard> {
       return Switch(
         onChanged: (val) {
           setState(() {
-            widget.value = !widget.value;
+            _value = !_value;
```

Comment:
ok so avoid creating stateful switches, most of the times such toggleables should be stateless and rely on the state of the host component